amazon_price_tracker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.9-slim
+WORKDIR /app
+COPY . /app
+RUN pip install --no-cache-dir beautifulsoup4 cabinet requests
```

Comment:
Thanks for your PRs. Please change to `RUN pip install --no-cache-dir -r requirements.md`